client/src/components/MapElement/MapElement.js

```diff
@@ -11,32 +13,88 @@ MapElement.propTypes = {
 export function MapElement({ className }) {
   const appView = useSelector((state) => state.view.appView)
   const activeTripId = useSelector((state) => state.view.activeTripId)
+  const dispatch = useDispatch()
+
+  const [destination, setDestination] = useState('')
+  const [stagesPerDay, setStagesPerDay] = useState('')
+  const [budget, setBudget] = useState('')
+  const [numberOfDays, setNumberOfDays] = useState('')
+  const [invalidForm, setInvalidForm] = useState(false)
+
+  const resetForm = () => {
+    setDestination('')
```

Comment:
Do we want to consider using react-hook-form?

Reply:
probably should, looks nicer